librecad/src/actions/rs_actiondimension.cpp

```diff
@@ -91,8 +91,6 @@ bool RS_ActionDimension::isDimensionAction(RS2::ActionType type) {
 	switch(type){
 	case RS2::ActionDimAligned:
 	case RS2::ActionDimLinear:
-	case RS2::ActionDimLinearVer:
-	case RS2::ActionDimLinearHor:
```

Comment:
What's the reason to remove these two useful tools?

Reply:
This PR is to merge ActionDimLinear, ActionDimLinearVer, ActionDimLinearHor features, by mouse movement.